saltgui/static/scripts/RunType.js

```diff
@@ -55,6 +60,8 @@ export class RunType {
     if (runType === undefined || runType === "") {
       return "normal";
```

Comment:
this should now be updated to `runType = "normal";`

Reply:
That makes sense and makes sure the initial runtype is saved due to no longer returning from the function prior.
Changed runType from `const` for obvious reasons.